R/helpers-ppc.R

```diff
@@ -250,6 +276,203 @@ all_counts <- function(x, ...) {
   all_whole_number(x, ...) && min(x) >= 0
 }
 
+#' Obtain the coverage parameter for simultaneous confidence bands for ECDFs
+#'
+#' @param N Length of sample.
+#' @param L Number of chains. Used for MCMC, defaults to 1 for ppc.
+#' @param K Number of equally spaced evaluation points (1:K / K). Defaults to N.
+#' @param prob Desired simultaneous coverage (0,1).
+#' @param M number of simulations to run, if simulation method is used.
+#' @param adj_method String defining the desired adjustment method. By default
+#'  "interpolate" is used. Other available options "optimize" and "simulate".
+#' @return The adjusted coverage parameter yielding the desired simultaneous
+#'  coverage of the ECDF traces.
+#' @noRd
+adjust_gamma <- function(N,
+                         L = 1,
+                         K = N,
+                         prob = 0.99,
+                         M = 1000,
+                         interpolate_adj = FALSE) {
+  if (! all_counts(c(K, N, L))) {
+    abort("Parameters 'N', 'L' and 'K' must be positive integers.")
+  }
+  if (prob >= 1 || prob <= 0) {
+    abort("Value of 'prob' must be in (0,1).")
+  }
+  if (interpolate_adj == TRUE) {
+    gamma <- interpolate_gamma(N, K, prob, L)
+  } else if (L == 1) {
+    gamma <- adjust_gamma_optimize(N, K, prob)
+  } else {
+    gamma <- adjust_gamma_simulate(N,
```

Comment:
I recommend naming arguments if there are more than 2 or 3 arguments given. Further, I recommend the format 

```R
function_name(
  name1 = input1,
  name2 = input2,
  ...
)
```

for function calls that are supposed to span multiple lines of code

Reply:
I went through the added functions and changed function calls to conform to this.